neuralmagic/benchmarks/configs/benchmark_serving_sparse.json

```diff
@@ -0,0 +1,36 @@
+{
+	"configs": [
+		{
+			"description": "Benchmark vllm serving",
+			"dataset_download_cmds": [
+				"wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json"
+			],
+			"models": [
+				"nm-testing/zephyr-50sparse-24"
+			],
+			"sparsity": [
```

Comment:
Note: we are going to remove the need for the user to have to specify sparsity in the cli launcher, so this will become unnecessary relatively soon

Reply:
Aha - Thanks for the heads up.